/**/**:
  ros__parameters:

    process_noise_gains:
      position: 0.1
      orientation: 0.01
      linear_velocity: 0.1
      angular_velocity: 0.1

    measurement_noise_gains:
      px4_odometry:
        position: 0.001
        orientation: 0.001
        linear_velocity: 0.01
        angular_velocity: 0.01
      openvins:
        position: 0.001
        orientation: 0.001
        linear_velocity: 0.01
        angular_velocity: 0.1
      fast_lio:
        position: 0.001
        orientation: 0.001
        linear_velocity: 0.1
        angular_velocity: 0.1
      imu:
        position: 0.1
        orientation: 0.1
        linear_velocity: 0.1
        angular_velocity: 0.01
      gps:
        position: 0.1
        orientation: 0.1
        linear_velocity: 0.1
        angular_velocity: 0.1